service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/PlacementApplicationContentProvider.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
+
+@Service
+public class PlacementApplicationContentProvider extends AbstractEmailContentProvider {
+    private final ObjectMapper mapper;
+
+    @Autowired
+    public PlacementApplicationContentProvider(@Value("${ccd.ui.base.url}") String uiBaseUrl,
+                                              ObjectMapper mapper,
+                                              HearingBookingService hearingBookingService,
+                                              DateFormatterService dateFormatterService) {
+        super(uiBaseUrl, dateFormatterService, hearingBookingService);
+        this.mapper = mapper;
+    }
+
+    public Map<String, Object> buildPlacementApplicationNotificationParameters(CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        return ImmutableMap.of(
+            "respondentLastName", getFirstRespondentLastName(caseData.getRespondents1()),
+            "caseUrl", formatCaseUrl(uiBaseUrl, caseDetails.getId())
```

Comment:
The [email template](https://www.notifications.service.gov.uk/services/12f756df-f01d-4a32-a405-e1ea8a494fbb/templates/ac0a0091-0b2e-44b9-afa1-321590b5d984) accepts `CaseUrl` but we are sending `caseUrl`. Is this case-insensitive?

Reply:
It must be as it's pulling it through, reverting template to use `caseUrl` for consistency with other templates regardless.